### DISCUSSION TITLE
Stats not current

### DISCUSSION THREAD

**Discussion by jb-hughes** [anuraghazra/github-readme-stats#1132] [Q&A] [answered]
Love the stats monitor in my profile: https://github.com/jb-hughes

The issue, as you may notice, is the stats aren't current, and are displaying my numbers as if I'm a GH newbie (I'm on my 3rd year in my current position using GH). Is this normal/ SOP?

**Comment by florianbussmann** [comment, marked as answer]
### checking your stats
 
Your stats card says 0 pull requests while the activity overview on your GitHub profile states 57% of your recent activity had been pull requests.. most likely they are on private repositories?

I see you used `count_private=true` for your stats card, but there was a discussion earlier that PR numbers are wrong if you don't deploy `github-readme-stats` yourself. You can read about it here https://github.com/anuraghazra/github-readme-stats/issues/1055#issuecomment-839187666. I hope that helps getting your stats right.

### ranking algorithm

Regarding the rank evaluation see this statement from the author https://github.com/anuraghazra/github-readme-stats/issues/455#issuecomment-757869757

**Reply by jb-hughes** [reply]
Thanks @florianbussmann ! Looks like your solution worked for PRs but the commits are now inaccurate. Does it take time for the APIs to sync? Am I missing something? https://github.com/jb-hughes

**Reply by rickstaa** [reply]
@jb-hughes For future reference, we do use a cache for updating the cards. More information can be found inhttps://github.com/anuraghazra/github-readme-stats#customization.
